images/kubectl-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -661,26 +661,7 @@ set -x
 ##############################################
 
 if [ "${LAGOON_PREROLLOUT_DISABLED}" != "true" ]; then
```

Comment:
I'm just thinking, we could actually remove anything to do with `LAGOON_PREROLLOUT_DISABLED` and `LAGOON_POSTROLLOUT_DISABLED` and have the build tool actually check these values (because the build tool is already evaluating the environments variables, it wouldn't be hard to bundle this into the tool too.

We'd then be able to remove this block of bash too https://github.com/uselagoon/lagoon/blob/5776aa8a86135957494f601b3df2d6de718bb501/images/kubectl-build-deploy-dind/build-deploy-docker-compose.sh#L508-L518